src/client/src/app/site/[site]/appointment/[reference]/cancel/cancel-appointment-page.test.tsx

```diff
@@ -5,7 +5,11 @@ import { useRouter } from 'next/navigation';
 import * as appointmentsService from '@services/appointmentsService';
 import render from '@testing/render';
 import { verifySummaryListItem } from '@components/nhsuk-frontend/summary-list.test';
-import { clinicalServices } from '@types';
+import { ClinicalService } from '@types';
+
+const clinicalServices: ClinicalService[] = [
+  { label: 'RSV Adult', value: 'RSV:Adult' },
+];
```

Comment:
Rather than repeat this block 8 times I'd recommend exporting it from the @testing folder as mockClinicalServices